### PR TITLE
Awaiting pronouncement with hearing date

### PR DESCRIPTION
**Pull request by dtaylor7**
[DIV-5028 - Landing page content for CoE (modify content)](https://tools.hmcts.net/jira/browse/DIV-5028)
  - Add new page for awaiting pronouncement with hearing date

### CODE REVIEW

File: steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.awaitingPronouncement.template.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<div class="panel panel-border-wide">
+    <p>{{ content.acceptedStatusPanelDetails }}</p>
+</div>
+
+<h2 class="heading-medium">{{ content.acceptedHearingTitle }}</h2>
+
+<p>{{ content.acceptedHearingMsg1 | safe }} </p>
+<div class="notice">
+    <em class="icon icon-important">
+        <span class="visually-hidden">Warning</span>
```
Comment (yarpo) [inline steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.awaitingPronouncement.template.html]: Shouldn't we also have it as a var? (for Welsh in the future)?

Comment (yarpo) [reply steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.awaitingPronouncement.template.html]: I don't mind to add it even in a sep PR, but please do not forget about it.

Comment (dtaylor7) [reply steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.awaitingPronouncement.template.html]: 👍

File: steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.content.awaitingPronouncement.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "acceptedStatusDetails": "A district judge will formally announce the decree nisi at a court hearing. This is the first stage of the decree ending your marriage.",
+    "acceptedStatusPanelDetails": "You'll only be divorced when the second stage is complete and you have your decree absolute.",
+    "acceptedHearingTitle": "The hearing",
+    "acceptedHearingMsg1": "You can find more details about the date of the hearing and the orders that will be made (including costs orders) in your <a href=\"{{ certificateOfEntitlementFile.uri | safe }}\" target=\"_blank\" aria-label=\"This link will download the Certificate Of Entitlement PDF.\">Certificate Of Entitlement</a> ({{ certificateOfEntitlementFile.fileType | upper }}).",
+    "acceptedHearingInfo": "You don’t need to come to the hearing unless you want to object to any of the court orders being made.",
+    "acceptedHearingMsg2": "If you want to attend the hearing, you need to email the court before the hearing date. If you, or your husband, want to dispute a costs order, the court must receive a letter or email stating the reasons for doing so at least 14 days before the hearing date."
```
Comment (alexwakeman) [inline steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.content.awaitingPronouncement.json]: We have hardcoded `**your husband**` which isn't right I think?

Comment (dtaylor7) [reply steps/petition-progress-bar/sections/awaitingPronouncement/PetitionProgressBar.content.awaitingPronouncement.json]: nice catch 👍

**Pull request merged**